package.json

```diff
@@ -17,5 +17,9 @@
   },
   "dependencies": {
     "rxjs": "^5.0.2"
+  },
+  "devDependencies": {
+    "react-native": "^0.43.2",
```

Comment:
Haha I see, I didn't know npm-link.

So, should we add react-native & react-native-windows to the peerDependencies ?

And, an another question, what if, for example the plugin is used so for android & ios, and not for windows, the project which include the plugin should also integrate the react-native-windows plugin?